### PR TITLE
Update docs/embeddings.md

### PR DESCRIPTION
**Pull request by kevinlu1248**
Copied from https://github.com/kevinlu1248/chroma-docs/pull/3.

## Description
This PR adds documentation for the HuggingFace embeddings in the `embeddings.md` and `usage-guide.md` files.

## Changes Made
- Added a new section in `embeddings.md` to document the HuggingFace embeddings. Explained the `HuggingFaceEmbeddingFunction` class and its methods. Provided information about the required `api_key` and `model_name` parameters. Included a link to the HuggingFace API for more information.
- Added a new example in `usage-guide.md` to demonstrate how to use the HuggingFace embeddings. Showed how to instantiate the `HuggingFaceEmbeddingFunction` class and generate embeddings.

## Checklist
- [ ] Updated the documentation in `embeddings.md`.
- [ ] Added a new example in `usage-guide.md`.

### CODE REVIEW

**Comment by atroyn** [comment]
Thanks @kevinlu1248. No need to change `usage-guide.md`, but the changes to `embeddings.md` look great!

File: docs/usage-guide.md [hunk 1]
```diff
@@ -191,6 +191,34 @@ Chroma uses collection names in the url, so there are a few restrictions on nami
 - The name must not contain two consecutive dots.
 - The name must not be a valid IP address.
 
+## HuggingFace Embeddings
```
Comment (atroyn) [inline docs/usage-guide.md:194]: I think we'd prefer to keep this section clean, without bringing up HF embeddings. Let's revert the change to this file, but keep the others.

**Review by atroyn — CHANGES_REQUESTED** [review]

**Comment by kevinlu1248** [comment]
Fixed @atroyn. We have a feature in Sweep where commenting "revert" on a file on the Sweep-created PR reverts all changes for that file.

**Review by atroyn — APPROVED** [review]
